language/move-vm/runtime/src/interpreter.rs

```diff
@@ -614,6 +674,11 @@ impl CallStack {
     fn pop(&mut self) -> Option<Frame> {
         self.0.pop()
     }
+
+    fn current_location(&self) -> Location {
```

Comment:
I find this odd, but I assume it was the only way to do it and we can review later.
In other words it seems strange to ask the call stack for the location

Reply:
Why? The call stack is by its nature what defines the location. In order to know what module/script I am in, I need to know what function I am in. The call stack is the source of truth for that information

Reply:
I did not see the details of the PR and maybe there are places where that matters.
I guess I was coming from a model where we are either inside a frame, and so unwinding would lead you there, or not and you don't need it, or that is what it seemed to me
Again I'll have to play with it when I merge ;) so I'll understand things better